File: Python/palindrome.py
n=int(input())
k = n
rev = 0
  
while(n > 0):
    
    a = n % 10
    rev = rev * 10 + a 
    n = n // 10
      
# print(rev) 

if(k==rev):
    print("true")
else:
    print("false")


n=int(input())
rev=0

k=n

while(n>0):
    a= n%10
    rev= rev*10+a
    n=n//10
    
if(rev==k):
    print("")

File: C/Palindrome.c
#include<stdio.h>
#include<stdbool.h>

bool isPalindrome(int a)
{
    int backup = a,rev = 0;

    while(a)
    {
        rev = (rev*10) + (a%10);
        a/=10;
    }

    return rev == backup;
}

int main()
{
    int n = 122;

    if(isPalindrome(n))
    printf("%d is a Palindrome Number",n);
    else
    printf("%d is not a Palindrome Number",n);

    return 0;

}
